scripts/README.md

```diff
@@ -0,0 +1,50 @@
+# Vagrant Kubernetes Cluster
```

Comment:
Documentation about the different groups of scripts.  This README is created to look the same as the other ones in the project - consistency